src/js/themes/base.d.ts

```diff
@@ -952,6 +956,77 @@ export interface ThemeType {
     colors?: GraphColorsType;
     extend?: ExtendType;
   };
+  pagination?: {
+    button?: {
```

Comment:
If the button type will be changed in the future, we might not be mindful to change the type instance on the pagination component as well, in order to avoid that, can we reuse the button type in a generic way so there will be one reusable type instance?

Reply:
Good question. I believe our thinking here was that we are only allowing customization of background colors and not supporting other customizations such as border. Would we be mis-setting expectations if we used `ButtonKindType`?

Reply:
Understood, thanks for clarifying. I think that if that's the case we should be good.